crates/sdk/src/keygen/mod.rs

```diff
@@ -193,6 +262,11 @@ impl AggStarkProvingKey {
             }
         });
         let internal_vm_vk = internal_vm_pk.vm_pk.get_vk();
+        assert!(check_recursive_verifier_size(
+            &internal_vm_vk,
+            config.internal_fri_params,
+            config.internal_fri_params.log_blowup
+        ));
```

Comment:
Let me know if these are the correct place to do these checks.

Reply:
prefer to make them all warnings (maybe with a helper function) instead of asserts so the sdk doesn't panic